src/components/VectorInput/index.js

```diff
@@ -174,6 +174,19 @@ class VectorInput extends Element {
     }
 
     set value(value) {
+        if (typeof value === 'string') {
```

Comment:
Should we try to parseInt(value, 10) first in case someone is passing a number as a string?